packages/Button/src/Button.styles.js

```diff
@@ -117,13 +117,15 @@ export const commonStyles = css`
   border-style: solid;
   border-width: 1px;
   box-sizing: border-box;
+  color: ${tokens.textColor.default};
```

Comment:
I must have forgotten to click `Add comment` before, but I remember writing that this line has inadvertently cause a regression in the `<Panel.Header>`.  The `<CloseButton>` was likely inheriting the white text colour from a parent, but now this overrides it as black. You can see the change on Screener. 

I think this change here is appropriate, but we should apply a styling override (or just propagate the `isDark` prop onto the `<CloseButton>`?).

Reply:
I wanted to check my advice locally to make sure it was good... and _sort of_.  

Seems you can remove these lines in the `<Panel>` and it should work properly again:

https://github.com/acl-services/paprika/blob/master/packages/Panel/src/components/Header/Header.styles.js#L48-L50